daemon/daemon_unix.go

```diff
@@ -189,6 +197,14 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 			return warnings, fmt.Errorf("Invalid value: %v, valid memory swappiness range is 0-100.", swappiness)
 		}
 	}
+	if hostConfig.KernelMemory > 0 && !CheckKernelVersion(4, 0, 0) {
```

Comment:
You mean fix in kernel? Actually it's not a bug, but a feature, support kernel memory reclaim so kernel memory limit is actually usable. I don't think it'll be backported to older kernel.

In kernel doc, kernel memory limit is an experimental feature before 4.0, maybe we can just warn here, to both daemon and client.